server/controller/schedulesController.js

```diff
@@ -0,0 +1,121 @@
+const conn = require("../mariadb");
+const { StatusCodes } = require("http-status-codes");
+
+const getScheduleByPeriod = ({ start, end }) => [
+  "SELECT * FROM SmartDay.schedules WHERE date BETWEEN ? AND ?",
+  [start, end],
+];
+
+const getScheduleByMonth = (month) => [
+  "SELECT * FROM SmartDay.schedules WHERE DATE_FORMAT(date, '%Y-%m') = ?",
+  [month],
+];
+
+const getScheduleByDate = (date) => ["SELECT * FROM SmartDay.schedules WHERE date = ?", [date]];
+
+const getSchedulesQuery = ({ start, end, date, month }) => {
+  if (start && end) return getScheduleByPeriod({ start, end });
```

Comment:
if문 같은 경우에는 문이 하나로 끝나더라도 중괄호를 사용해주는게 일반적으로 권장되기는 해요~
```suggestion
  if (start && end) {
   return getScheduleByPeriod({ start, end });
 }
```